test/test_torch.py

```diff
@@ -3553,7 +3553,9 @@ def test_index_fill(self, device, dtype):
     # FIXME: move to test indexing
     # The test fails for zero-dimensional tensors on XLA
     @onlyNativeDeviceTypes
-    @dtypes(*all_types_and_complex_and(torch.half, torch.bool, torch.bfloat16))
+    @dtypes(*all_types_and_complex_and(torch.half, torch.bool, torch.bfloat16,
+                                       torch.float8_e4m3fn, torch.float8_e4m3fnuz,
```

Comment:
optional nit: maybe save the float8 types in a list and unpack everywhere? will make it easier for future op support PRs to not forget to handle all of them